src/System.Net.Mail/src/System/Net/Mail/SmtpAccess.cs

```diff
@@ -9,65 +9,8 @@ namespace System.Net.Mail
 {
     public enum SmtpAccess { None = 0, Connect = 1, ConnectToUnrestrictedPort = 2 };
 
-    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Constructor | AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Assembly, AllowMultiple = true, Inherited = false)]
     [Serializable]
-    public sealed class SmtpPermissionAttribute : CodeAccessSecurityAttribute
```

Comment:
This is not exposed in the netstandard2.0 contract, and not being used internally, so deleted. I missed this while porting the initial set of APIs, exposed these additional members.